### PR TITLE
Coreboot 4.20 contrib

### PR DESCRIPTION
**Pull request by pietrushnic**

### CODE REVIEW

**Review by macpijan — APPROVED** [review]

File: blog/content/post/2023-05-18-our_contribution_to_coreboot_4_20_release.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+---
+title: Our contribution to coreboot 4.20 release
+abstract: '🎉 Celebrating coreboot 4.20 release! 🚀 Kudos to our contributors
+           who have pushed the envelope in firmware security & optimization. Key updates:
+           improved SoC support, TPM security, VT-d DMA protection & more. Check out the
+          blog for all the details.'
```
Comment (miczyg1) [inline blog/content/post/2023-05-18-our_contribution_to_coreboot_4_20_release.md:6]:
```suggestion
           blog for all the details.'
```

File: blog/content/post/2023-05-18-our_contribution_to_coreboot_4_20_release.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+* VT-d: The VT-d subsystem now has a new DMA protection API, which we integrated into Alder Lake's functionality.
+* EDK2 Payload: Users can now clone the edk2-platforms repository.
+* Bug Fixes: Multiple fixes in different areas, such as Intel Elkhart Lake's GPIO and Makefiles.
+* Power9: We refactored the code to enhance readability and maintainability.
+* Additional features: A speaker beep function and updated USB port macros.
+
+## Contribution details
+
+* Kacper Stojek (3)
+	* [mainboard/protectli/vault_ehl: Add initial structure ](https://review.coreboot.org/c/coreboot/+/72407)
```
Comment (macpijan) [inline blog/content/post/2023-05-18-our_contribution_to_coreboot_4_20_release.md:76]: extra spaces in some links descriptions
also, we should use spaces probably, not tabs?

File: blog/content/post/2023-05-18-our_contribution_to_coreboot_4_20_release.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+3mdeb has actively contributed to this release as part of [Dasharo Support
+Package](https://docs.dasharo.com/osf-trolling-list/jsm_documentation/#dasharo-support-package)
+product and [Dasharo Community
+Support](https://docs.dasharo.com/osf-trivia-list/dasharo/#dasharo-professional-support)
+sponsored through [Dasharo newsletter
+subscription](https://3mdeb.com/?s=%22year+Dasharo+Supporters+Entrance%22&post_type=product&dgwt_wcas=1)
+and [merchandise](https://3mdeb.com/product-category/merchandise/), as seen in
+the number of commits, mainly through our work in:
+
+* Mainboard and SoC Support: We added and improved support for Protectli mainboards based on Intel Elkhart Lake, Alder Lake, and Jasper Lake SoCs.
```
Comment (miczyg1) [inline blog/content/post/2023-05-18-our_contribution_to_coreboot_4_20_release.md:64]:
```suggestion
* Mainboard and SoC Support: We added and improved support for Protectli mainboards based on Intel Elkhart Lake, Alder Lake SoCs. Added support to dump GPIOs on Jasper Lake SoC.
```

There are no Protectli mainboards based on Jasper Lake SoC which we improved. We have simply added a possibility to dump GPIOs on Jasper Lake SoC

File: blog/content/post/2023-05-18-our_contribution_to_coreboot_4_20_release.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+* Documentation: coreboot's documentation was updated, particularly in the Dasharo description and Trusted Platform Module (TPM) options.
+* TPM Security: New log formats compliant with the 2.0 and 1.2 specs and Kconfig-configurable PCR usage.
+* VT-d: The VT-d subsystem now has a new DMA protection API, which we integrated into Alder Lake's functionality.
+* EDK2 Payload: Users can now clone the edk2-platforms repository.
+* Bug Fixes: Multiple fixes in different areas, such as Intel Elkhart Lake's GPIO and Makefiles.
+* Power9: We refactored the code to enhance readability and maintainability.
+* Additional features: A speaker beep function and updated USB port macros.
```
Comment (miczyg1) [inline blog/content/post/2023-05-18-our_contribution_to_coreboot_4_20_release.md:71]: Not wrapped at 80

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: blog/content/post/2023-05-18-our_contribution_to_coreboot_4_20_release.md (hunk 4 above)
Comment (miczyg1) [reply blog/content/post/2023-05-18-our_contribution_to_coreboot_4_20_release.md]: Fixed

**Review by miczyg1 — APPROVED** [review]

**Pull request merged**